CMakeLists.txt

```diff
@@ -307,41 +307,41 @@ else(WIN32)
+src/mappostgresql.c src/mapthread.c src/mapcopy.c src/maplabel.c src/mapprimitive.c src/maptile.c
+src/mapcpl.c src/maplayer.c src/mapproject.c src/maptime.c src/mapcrypto.c src/maplegend.c src/hittest.c
+src/maptree.c src/mapdebug.c src/maplexer.c src/mapquantization.c src/mapunion.cpp
+src/mapdraw.c src/maplibxml2.c src/mapquery.c src/maputil.c src/strptime.c src/mapdrawgdal.c
+src/mapraster.c src/mapuvraster.c src/mapdummyrenderer.c src/mapobject.c src/maprasterquery.c
+src/mapwcs.cpp src/maperror.c src/mapogcfilter.cpp src/mapregex.c src/mapwcs11.cpp src/mapfile.c
+src/mapogcfiltercommon.cpp src/maprendering.c src/mapwcs20.cpp src/mapogcsld.c src/mapmetadata.c
+src/mapresample.c src/mapwfs.cpp src/mapgdal.cpp src/mapogcsos.c src/mapscale.c src/mapwfs11.cpp src/mapwfs20.c
+src/mapgeomtransform.c src/mapogroutput.cpp src/mapwfslayer.c src/mapagg.cpp src/mapkml.cpp
+src/mapgeomutil.cpp src/mapkmlrenderer.cpp src/fontcache.c src/textlayout.c src/maputfgrid.cpp
+src/mapogr.cpp src/mapcontour.c src/mapsmoothing.c src/mapv8.cpp ${REGEX_SOURCES} src/kerneldensity.c
+src/idw.c src/interpolation.c src/mapflatgeobuf.c src/mapcompositingfilter.c src/mapmvt.c src/mapiconv.c
+src/mapgraph.cpp src/mapserv-config.cpp)
 
 set(mapserver_HEADERS
-cgiutil.h dejavu-sans-condensed.h dxfcolor.h fontcache.h hittest.h mapagg.h
-mapaxisorder.h mapcopy.h mapentities.h maperror.h mapfile.h mapgml.h maphash.h
-maphttp.h mapio.h mapkmlrenderer.h maplibxml2.h mapogcfilter.h mapogcsld.h
-mapoglcontext.h mapoglrenderer.h mapowscommon.h mapows.h mapparser.h mapogcapi.h
-mappostgis.h mapprimitive.h mapproject.h mapraster.h mapregex.h mapresample.h
-mapserver-api.h mapserver.h mapserv.h mapshape.h mapsymbol.h maptemplate.h
-mapthread.h maptile.h maptime.h maptree.h maputfgrid.h mapwcs.h uthash.h mapiconv.h
-mapflatgeobuf.h mapgraph.h mapserv-config.h)
+src/cgiutil.h src/dejavu-sans-condensed.h src/dxfcolor.h src/fontcache.h src/hittest.h src/mapagg.h
+src/mapaxisorder.h src/mapcopy.h src/mapentities.h src/maperror.h src/mapfile.h src/mapgml.h src/maphash.h
+src/maphttp.h src/mapio.h src/mapkmlrenderer.h src/maplibxml2.h src/mapogcfilter.h src/mapogcsld.h
+src/mapoglcontext.h src/mapoglrenderer.h src/mapowscommon.h src/mapows.h src/mapparser.h src/mapogcapi.h
+src/mappostgis.h src/mapprimitive.h src/mapproject.h src/mapraster.h src/mapregex.h src/mapresample.h
+src/mapserver-api.h src/mapserver.h src/apps/mapserv.h src/mapshape.h src/mapsymbol.h src/maptemplate.h
+src/mapthread.h src/maptile.h src/maptime.h src/maptree.h src/maputfgrid.h src/mapwcs.h src/uthash.h src/mapiconv.h
+src/mapflatgeobuf.h src/mapgraph.h src/mapserv-config.h)
 
 if(WIN32)
 	configure_file(
-	  ${CMAKE_CURRENT_SOURCE_DIR}/version.rc.in
-	  ${CMAKE_CURRENT_BINARY_DIR}/version.rc
+	  ${CMAKE_CURRENT_SOURCE_DIR}/src/version.rc.in
+	  ${CMAKE_CURRENT_BINARY_DIR}/src/version.rc
 	  @ONLY)
 	set(mapserver_SOURCES ${mapserver_SOURCES} ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
```

Comment:
@jmckenna - I think this also needs to be updated to:

`set(mapserver_SOURCES ${mapserver_SOURCES} ${CMAKE_CURRENT_BINARY_DIR}/src/version.rc)`

See errors in https://ci.appveyor.com/project/MapServer/mapserver/builds/47765494/job/fw9usno6l5315808

Reply:
good catch, thanks @geographika